setup.py

```diff
@@ -203,6 +203,7 @@
             'inspire_cache = inspirehep.modules.cache.ext:INSPIRECache',
             'inspire_search = inspirehep.modules.search:INSPIRESearch',
             'inspire_workflows = inspirehep.modules.workflows:INSPIREWorkflows',
+            'invenio_collections = invenio_collections:InvenioCollections',
```

Comment:
Sorry, how come? You should be testing the PUT request here: https://github.com/inspirehep/inspire-next/pull/1678/files#diff-8eedb14f28e016fe0e021789e25878abR348

If something else is failing unless you have this line then we're missing a test.